apps/web/src/components/Explore/index.tsx

```diff
@@ -45,49 +44,43 @@ const Explore: NextPage = () => {
         title={`Explore • ${APP_NAME}`}
       />
       <GridItemEight className="space-y-5">
-        <Tab.Group
-          defaultIndex={Number(router.query.tab)}
-          onChange={(index) => {
+        <Tabs.Root
+          onValueChange={(index) => {
             router.replace(
-              { query: { ...router.query, tab: index } },
+              { query: { ...router.query, tab: parseInt(index) } },
               undefined,
               { shallow: true }
             );
           }}
+          value={`${Number(router.query.tab ?? 0)}`}
         >
-          <Tab.List className="divider space-x-8">
+          <Tabs.List className="divider space-x-8">
             {tabs.map((tab, index) => (
-              <Tab
-                className={({ selected }) =>
-                  cn(
-                    {
-                      'border-brand-500 border-b-2 !text-black dark:!text-white':
-                        selected
-                    },
-                    'ld-text-gray-500 px-4 pb-2 text-xs font-medium outline-none sm:text-sm'
-                  )
+              <Tabs.Trigger
+                className={
+                  'ld-text-gray-500 px-4 pb-2 text-xs font-medium outline-none sm:text-sm data-[state=active]:border-brand-500 data-[state=active]:border-b-2 data-[state=active]:!text-black dark:data-[state=active]:!text-white'
                 }
-                defaultChecked={index === 1}
                 key={tab.type}
                 onClick={() => {
                   Leafwatch.track(EXPLORE.SWITCH_EXPLORE_FEED_TAB, {
                     explore_feed_type: tab.type.toLowerCase()
                   });
                 }}
+                value={`${index}`}
               >
                 {tab.name}
-              </Tab>
+              </Tabs.Trigger>
             ))}
-          </Tab.List>
-          <FeedFocusType focus={focus} setFocus={setFocus} />
-          <Tab.Panels>
-            {tabs.map((tab) => (
-              <Tab.Panel key={tab.type}>
-                <Feed feedType={tab.type} focus={focus} />
-              </Tab.Panel>
-            ))}
-          </Tab.Panels>
-        </Tab.Group>
+          </Tabs.List>
+          <div className={'mt-4'}>
+            <FeedFocusType focus={focus} setFocus={setFocus} />
+          </div>
+          {tabs.map((tab, index) => (
+            <Tabs.Content className={'mt-4'} key={tab.type} value={`${index}`}>
```

Comment:
why we need mt-4 also it should not wrapped under `{}`

Reply:
Tabs.Content and Tabs-Content in radix ui doesn't have a default margin set. That's why added margin to keep the look of UI intact with the previous one.